System.IO.Abstractions.TestingHelpers.Tests/MockDirectoryTests.cs

```diff
@@ -1182,6 +1182,27 @@ public void MockDirectory_Move_ShouldMove(string sourceDirName, string destDirNa
             Assert.IsTrue(fileSystem.File.Exists(XFS.Path(destDirName + filePathTwo)));
         }
 
+        [Test]
+        public void MockDirectory_Move_ShouldMoveFiles()
+        {
+            string sourceFilePath = XFS.Path(@"c:\something\demo.txt");
+            string sourceFileContent = "this is some content";
+            
+            var fileSystem = new MockFileSystem(new Dictionary<string, MockFileData>
+            {
+                { sourceFilePath, new MockFileData(sourceFileContent) },
+                { XFS.Path(@"c:\somethingelse\dummy.txt"), MockFileData.NullObject }
+            });
+
+            string destFilePath = XFS.Path(@"c:\somethingelse\demo1.txt");
+
+            fileSystem.Directory.Move(sourceFilePath, destFilePath);
+
+            Assert.That(fileSystem.FileExists(destFilePath), Is.True);
+            Assert.That(fileSystem.GetFile(destFilePath).TextContents, Is.EqualTo(sourceFileContent));
+            Assert.That(fileSystem.FileExists(sourceFilePath), Is.False);
```

Comment:
I'm usually against multiple asserts, but since this is more about state after a move, it seems OK. I had a slight issue with this because it was initially kinda confusing. Maybe we move the FileExists check to the top so we're validating state together? i.e. FileExists destFile (true) and FileExists sourceFile (false) need to be together to validate the file moved on the file system.  The GetFile makes sure the source and dest were the same.

Reply:
Good catch, will do